composer.json

```diff
@@ -38,9 +38,10 @@
         "swiftmailer/swiftmailer": "^6.2",
         "symfony/cache": "^3.4 || ^4.0",
         "symfony/config": "^3.4 || ^4.0",
+        "symfony/finder": "^3.4 || ^4.0",
```

Comment:
This should be removed, I've added it just to prevent 3.2 from being installed, which is possibly causing email tests to fail on https://app.travis-ci.com/github/silverstripe/silverstripe-framework/jobs/559084533

If it turns out this fixed it, silverstripe/config should be updated to change ^3.2 to ^3.4